### DISCUSSION TITLE
vaultwarden just hanging not even a login screen

### DISCUSSION THREAD

**Discussion by bernoussama** [dani-garcia/vaultwarden#4760] [Q&A]
[Screencast from 07-17-2024 02:05:43 AM.webm](https://github.com/user-attachments/assets/cb94d8c2-10c0-4710-899c-cc73788552de)

also updated to latest version, removed the container and reran it . and still hanging
here's the logs:

![image](https://github.com/user-attachments/assets/5b124c9d-031f-4368-a36b-ec686bcf65da)

**Comment by BlackDex** [comment]
If that is all you see in the logs then something strange is going on.
Which version are  you running? Debian or Alpine? If it is Debian, i would suggest to use the alpine version and see what happens.

Also try to set the log level to debug `LOG_LEVEL=debug` and see if that provides more info.

And verify that your reverse proxy (caddy) isn't providing cached versions of old files, since the web-vault was updated and that also might cause issues.

**Reply by bernoussama** [reply]
I'm using the default image not sure what is it based on, anyways it fixed itself the day after, probably a caching issue because it does hang from from time to time. I'm also proxying it through Cloudflare tunnels, so it's either Caddy or Cloudflare caching that's causing the issue. Will look into it.
Thank you, and sorry for my late response.

**Reply by stefan0xC** [reply]
You might also want to check your available disk space. If the disk is full systems tend to fail in unexpected ways.

**Reply by bernoussama** [reply]
> You might also want to check your available disk space. If the disk is full systems tend to fail in unexpected ways.

Yeah, but already checked that, the disk still has plenty of space.

**Reply by TheekshanaA** [reply]
> > You might also want to check your available disk space. If the disk is full systems tend to fail in unexpected ways.
> 
> Yeah, but already checked that, the disk still has plenty of space.

Did you manage to get this resolved?

**Reply by skylabby** [reply]
Has anyone manage to solve this issue.

**Reply by TheekshanaA** [reply]
I managed to resolve the issue by adding an SSL certificate to my vaultwarden server. I used nginx proxy manager.

**Reply by St-Ilya** [reply]
@TheekshanaA 
can you please send all your configs or the order of execution actions?

**Reply by TheekshanaA** [reply]
@St-Ilya https://youtu.be/P3imFC7GSr0 Follow this and you'll need to create a local CA and issue a cert for vaultwarden unless you plan on opening it up to the internet then you can use the built-in Letsencrypt to automatically generate a cert for vaultwarden.

**Comment by devZer0** [comment]
do you access https:// or http:// url with the browser?

you need to enable https first, as browser won't connect otherwise. 

https://github.com/dani-garcia/vaultwarden/discussions/5549
